pagopa/io-app
# feat: [PE-610] CGN detail screen new card appearance

## Short description
Use new appearance for card in CGN card details screen.
The change is merely in the graphics of the header as you can see in the figma.
All the other aspects should have remained the same

## List of changes proposed in this pull request
- remove component CgnCardComponent in favor of using existing BonusCardScreenComponent
- fix tests by extracting CgnCardStatus and IdPayStatus component
- updated call sites at IdPayInititativeDetailScreen and CgnDetailScreen
- add prop footer to BonusCard component
- add prop secondary cat to BonusCard component
- refactor prop status in BonusCard component
- ported animated background into CgnDetailScreen file

## How to test
[figma](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6171-28555&t=Q93NgZDycQB70grv-4)
- login into the app
- go into the wallet
- open "Carta giovani nazionale"
- check the new appearance of the animated header here, and ensure the rest of the page did not change
